samples/nrf9160/coap_client/src/main.c

```diff
@@ -132,7 +132,12 @@ static int client_handle_get_response(uint8_t *buf, int received)
 		return 0;
 	}
 
-	snprintf(temp_buf, MAX(payload_len, sizeof(temp_buf)), "%s", payload);
+	if(payload_len > 0) {
+		snprintf(temp_buf, MAX(payload_len, sizeof(temp_buf)), "%s", payload);
+	}
+	else {
+		strcpy(temp_buf, "EMPTY PAYLOAD");
```

Comment:
Could we make it just "EMPTY", it should align nicely with the log message below (`payload: EMPTY`).